modules/340-monitoring-kubernetes/hooks/choose_nodes_for_ebpf_exporter.go

```diff
@@ -78,10 +101,27 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 			Kind:       "Node",
 			FilterFunc: getNodeNameWithSupportedDistro,
 		},
+		{
+			Name:       "nodegroups",
+			ApiVersion: "deckhouse.io/v1",
+			Kind:       "NodeGroup",
+			FilterFunc: nodeGroupHasManagedKernel,
```

Comment:
We decided to allow run ebpf exporter on nodes even if kernel is unmanaged. It's better to use entrypoint instead of hook.

Reply:
Done!